lib/bond/generated/CsProtocol_readers.hpp

```diff
@@ -315,14 +310,14 @@ bool Deserialize(TReader& reader, ::CsProtocol::Device& value, bool isBase)
                 }
                 break;
             }
-#ifdef HAVE_CS4
+
```

Comment:
Why is the check of HAVE_CS4 removed?

Reply:
Because I re-ran the bond compiler to generate new classes with the updated fields, and I did not realize we had manually modified this generated file. I'll need to udpate this.